client/pages/resetPassword.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
```

Comment:
I think we should avoid capitalization in our route names, I'm not sure how to name this page though...

maybe /login/forgot.vue or /login/reset.vue

making the current login page /login/index.vue (so the actual route wont change)

Reply:
btw dont forget to change all the references to /resetPassword

Reply:
How about `/reset-password`?

Reply:
is it common to have dashes in non dynamic routes ? if so, why not i guess

Reply:
According to multiple sources, including [this one](https://restfulapi.net/resource-naming/), it is quite the right choice to use snake case.

Reply:
kebab*, aight fair enough, let's settle for reset-password